homeassistant/components/xiaomi_aqara/__init__.py

```diff
@@ -33,6 +33,7 @@
 CONF_INTERFACE = "interface"
 CONF_KEY = "key"
 CONF_DISABLE = "disable"
+CONF_PROTO = "proto"
```

Comment:
Use `protocol` or `protocol_version` instead of shorthand.

Reply:
Unfortunately, the xiaomi_gateway used by this module expects `proto`, so it is not in my control to use a particular string.